packages/core/src/PasteRule.ts

```diff
@@ -164,13 +164,25 @@ export function pasteRulesPlugin(props: { editor: Editor; rules: PasteRule[] }):
           dragSourceElement = view.dom.parentElement?.contains(event.target as Element)
             ? view.dom.parentElement
             : null
+
+          if (dragSourceElement) {
+            (window as any).tiptapDragFromOtherEditor = editor
```

Comment:
Do we need to save into the global window instance here? Is there no other way to keep track of the editor instance?

Reply:
> Do we need to save into the global window instance here? Is there no other way to keep track of the editor instance?

Because when dragging across editors, there are multiple editor instances. If you delete the selection content of another editor instance in the drop method of one editor instance, you must get a reference to another editor instance.

I can only simply store it on the window, unless tipap provides a similar global store or variable to store this kind of thing. @bdbch

Reply:
If you have a better way, we can modify it again.

Reply:
pushed again

https://github.com/ueberdosis/tiptap/pull/4328/commits/5b906e7456c3dc03ded851588a424dedaca58cf4